Makefile

```diff
@@ -102,6 +102,11 @@ build-benchmarks:
 	@cd benchmark/performanceTest ; GO111MODULE=$(GO111MODULE_VALUE) go build -o ../bin/PerfTests .
 	@cd benchmark/comparisonTest ;  GO111MODULE=$(GO111MODULE_VALUE) go build -o ../bin/CompTests .
 
+benchmarks-test:
+	@echo "$@"
+	@cd benchmark/performanceTest ; sudo rm -rf output ; GO111MODULE=$(GO111MODULE_VALUE) go build -o ../bin/PerfTests . && sudo ../bin/PerfTests -show-commit
+	@cd benchmark/performanceTest ; cat output/results.json
```

Comment:
For consistency we should have the `benchmark` workflow cat the results  similar to to the comparison test workflow.

Reply:
Done